.github/workflows/build.yml

```
@@ -0,0 +1,57 @@
+name: Build images
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'eureka-relayer-v[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch: # this will kick off a dev deployment
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+    env:
+      registry_url: ghcr.io/cosmos/eureka-relayer
+    steps:
+      - name: Check out the repo
+        if: github.event_name != 'pull_request'
+        uses: actions/checkout@v4
+      - name: Check out the PR commit head
+        uses: actions/checkout@v4
+        if: github.event_name == 'pull_request'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        env:
+          DOCKER_METADATA_PR_HEAD_SHA: true
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ${{ env.registry_url }}
+          tags: |
+            type=sha,priority=900,prefix=
+            type=semver,priority=1000,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
```

What's the difference between this two, shouldn't we only have the top one? It seems that if we tag a release with `v1.2.3`. Then the top one will map to `v1.2.3` and the other to `1.2.3`?

```suggestion
            type=semver,priority=1000,pattern={{version}}
```